x-pack/plugins/ml/public/jobs/jobs_list/components/jobs_list_view/jobs_list_view.js

```diff
@@ -247,7 +248,14 @@ export class JobsListView extends Component {
 
   refreshJobSummaryList(forceRefresh = false) {
     if (forceRefresh === true || this.blockRefresh === false) {
+
+      // Set loading to true for jobs_list table for initial job loading
+      if (this.state.loading === null) {
```

Comment:
as this `refreshJobSummaryList` is called at the beginning and loading is never reverted back to `null`, could `loading`'s initial state just be `true`? 
this check could then be dropped.

Reply:
actually, I've changed my mind. I think it's ok to have `loading` in an uninitialised state at he beginning.
I'll leave my comment here anyway.